src/less/page-notice/page-notice.stories.js

```diff
@@ -8,11 +8,28 @@ export const confirmation = () => `
         </svg>
     </div>
     <div class="page-notice__main">
-        <h3 class="page-notice__title">Your new listing is live:</h3>
-        <p><a href="#">Spam and Eggs From the Cow's Perspective</a> (paperback).</p>
+        <h3>You have opted into eBay Pay and </h3>
```

Comment:
It is not correct to split a sentence across two heading tags. This will break heading structure and create a poor/confusing reading experience.